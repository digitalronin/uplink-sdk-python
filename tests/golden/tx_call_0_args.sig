r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 72377614485416968158335204094331140694372965924444226058821280446306833273768
