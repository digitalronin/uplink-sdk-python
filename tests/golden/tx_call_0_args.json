{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE02OTY5ODgzODEzNDA0NDM0NTU3NTE5ODAwNTM3NzE2NDg5NjgxNjU0ODYwNzE1MjEzOTM2NDkwMDQ2NTYxMDk5ODg1MTUzNTE2NjIxMw==", 
    "timestamp": 1231006505
}